src/battle_script_commands.c

```diff
@@ -8041,11 +8037,11 @@ static bool32 CanTeleport(u8 battlerId)
     switch (GetBattlerSide(battlerId))
     {
     case B_SIDE_OPPONENT:
-        if (gBattleTypeFlags & BATTLE_TYPE_DOUBLE)
+        if (count == 1 || gBattleTypeFlags & BATTLE_TYPE_DOUBLE)
             return FALSE;
         break;
     case B_SIDE_PLAYER:
-        if (count == 1 || (count <= 2 && gBattleTypeFlags & BATTLE_TYPE_DOUBLE))
+        if (count == 1 || (gBattleTypeFlags & BATTLE_TYPE_DOUBLE && count <= 2))
```

Comment:
Why change this order?

Reply:
Because my brain thought "If there's only one Pokémon left in the party or if it's a double battle and there's 2 Pokémon or less..." and then I just instinctively made the change :P

Speaking of which, I should change that to `< 2`, so let me know if you want me to roll it back so I do both in the same commit.

Reply:
> Speaking of which, I should change that to `< 2`

Wait, why? wouldn't having it as `count < 2` be essentially the same as `count == 1`?

Reply:
> Wait, why?

Nvm, I didn't notice that it's the check for doubles, lmao. `<= 2` is correct because if there's only 2 Pokémon alive that means there's no viable options to switch with.

So, want me to change it back from `(gBattleTypeFlags & BATTLE_TYPE_DOUBLE && count <= 2)` to `(count <= 2 && gBattleTypeFlags & BATTLE_TYPE_DOUBLE)` or it's good to go? :eyes:

Reply:
It's good to go, I also code in sentence order, so it's more readable to me the way you did :)